config/crds/apis.kcp.io_apiexports.yaml

```diff
@@ -489,12 +497,15 @@ spec:
                           type: object
                       type: object
                   required:
+                  - group
+                  - name
                   - schema
                   - storage
```

Comment:
this is wrong. Storage has a default, but is not required. A client verifying this client-side does not see the default and will fail.

Reply:
I don't understand. If this is defaulted and we need some sort of storage defined for a resource schema, how exactly is storage then not a required field? How do we deal with resource schemas without a storage? Which, by not making it a require field, you imply is possible and valid.

Reply:
`this is wrong. Storage has a default, but is not required` - not sure what this means? Should it be optional if it has default? or what?

Reply:
Can must not require defaulted fields. The client (kubectl or others) will fails with their client-side validation.

Reply:
reduced validation on `storage`